src/electron/electron/search.cljs

```diff
@@ -120,12 +121,15 @@
 
 (defn open-db!
   [db-name]
-  (let [[db-sanitized-name db-full-path] (get-db-full-path db-name)
-        db (sqlite3 db-full-path nil)
-        _ (create-blocks-table! db)
-        _ (create-blocks-fts-table! db)
-        _ (add-triggers! db)]
-    (swap! databases assoc db-sanitized-name db)))
+    (let [[db-sanitized-name db-full-path] (get-db-full-path db-name)]
+      (try (let [db (sqlite3 db-full-path nil)]
+             (create-blocks-table! db)
+             (create-blocks-fts-table! db)
+             (add-triggers! db)
+             (swap! databases assoc db-sanitized-name db))
+           (catch :default e
+             (.error logger (str e ": " db-name))
```

Comment:
May add instruction here? e.g.
Removing corrupted index cache. Please relaunch Logseq later.

Reply:
The corrupted DB is removed directly. 
The app will start usually and re-build search index automatically.